client/src/components/annotators/annotator.js

```diff
@@ -72,6 +72,28 @@ export default {
     setImageCursor(newCursor) {
       this.imageCursor = `${newCursor}`;
     },
+    resetMapDimensions() {
+      this.geoViewer.bounds({
+        left: 0,
+        top: 0,
```

Comment:
relates to above where we filter out the same width/height values.  So perhaps changing this to `resetMapDimensions(width, height)`, comparing with the current `this.width/this.height` and if it is the same just returning.